frontend/amundsen_application/static/js/ducks/feature/reducer.ts

```diff
@@ -86,6 +94,31 @@ export function getFeatureCodeFailure(
   };
 }
 
+export function getFeaturePreviewData(
+  payload: FeaturePreviewQueryParams
+): GetFeaturePreviewDataRequest {
+  return {
+    payload,
+    type: GetFeaturePreviewData.REQUEST,
+  };
+}
```

Comment:
No tracking?

Reply:
The preview gets loaded automatically when the page loads. We have tracking on the tab component.